GetSunsetRise.cpp

```diff
@@ -0,0 +1,198 @@
+#include "GetSunsetRise.h"
+
+#include <QAction>
+#include <QFile>
+#include <QMessageBox>
+#include <QStringList>
+#include <QTextStream>
+#include <QtScript/QScriptEngine>
+#include <QtScript/QScriptValue>
+
+GetSunsetRise::GetSunsetRise(QWidget *pwgt)
+    : QWidget(pwgt)
+{
+    setupUi(this);
+    QFile file(QString(":/timeZonesRC.txt"));
```

Comment:
You should avoid to load this, because it may changed and ui-file has Russian names. Make it all more independence.

Reply:
Done